tests/test_date.py

```diff
@@ -284,6 +284,12 @@ def test_should_parse_date_with_timezones_using_format(self):
         self.assertEqual('day', date_data['period'])
         self.assertEqual(expected, date_data['date_obj'])
 
+    def test_should_parse_with_no_break_space_in_dates(self):
+        date_string = "08-08-2014 18:29"
```

Comment:
It is not clear on reading that it is no-break space. Let's use explicit representation here.